test/runtests.jl

```diff
@@ -2,12 +2,20 @@
 using FFTW
 using Base.Test
 
-# Make super sure we don't pull in Base names
 importall FFTW
-import AbstractFFTs: Plan, fft, ifft, bfft, fft!, ifft!, bfft!,
+if VERSION >= v"0.7.0-DEV.602"
+    import AbstractFFTs: Plan, fft, ifft, bfft, fft!, ifft!, bfft!,
+                         plan_fft, plan_ifft, plan_bfft, plan_fft!, plan_ifft!, plan_bfft!,
+                         rfft, irfft, brfft, plan_rfft, plan_irfft, plan_brfft,
+                         fftshift, ifftshift, plan_inv
+    import FFTW: fftw_vendor
+else
+    import Base.DFT: Plan, fft, ifft, bfft, fft!, ifft!, bfft!,
                      plan_fft, plan_ifft, plan_bfft, plan_fft!, plan_ifft!, plan_bfft!,
```

Comment:
I just copied and pasted, but originally I had tried with just `Plan` but there was something else it was getting mad about (can't recall what it was offhand, `plan_` something or other)